atomic_reactor/plugins/post_group_manifests.py

```diff
@@ -46,6 +49,66 @@ def __init__(self, tasker, workflow, registries, group=True, goarch=None):
+        docker_push_secret = registry_conf.get('secret', None)
+        with NamedTemporaryFile(prefix='manifest-list', suffix=".yml", mode='w') as fp:
+            yaml.dump(manifest_list_spec, stream=fp)
+            fp.flush()
+            self.log.info("Wrote to file %s", fp.name)
+
+            cmd = ['manifest-tool', '--docker-cfg=%s' % docker_push_secret,
+                   'push', 'from-spec', fp.name]
+            subprocess.check_call(cmd)
+            self.log.info("Manifest list submitted")
+
+    def get_grouped_manifests(self):
+        grouped_manifests = []
+        for registry, registry_conf in self.registries.items():
+            manifest_list_spec = {}
+            manifest_list_spec['manifests'] = []
+            all_annotations = self.workflow.build_result.annotations['worker-builds']
+            for platform in all_annotations:
+                worker_image = all_annotations[platform]['digests'][0]
+                digest = worker_image['digest']
+                repository = worker_image['repository']
+                arch_entry = {
+                    'image': '{0}/{1}@{2}'.format(registry, repository, digest),
+                    'platform': {
+                        'os': 'linux',
+                        'architecture': self.goarch.get(platform, platform)
+                    }
+                }
+                manifest_list_spec['manifests'].append(arch_entry)
+
+            for image in self.workflow.tag_conf.images:
+                manifest_list_spec['image'] = image
+                self.log.info("Submitting manifest-list %s", manifest_list_spec)
+                self.submit_manifest_list(registry, registry_conf, manifest_list_spec)
+            insecure = registry_conf.get('insecure', False)
+            secret_path = registry_conf.get('secret')
+
+            registry_image = image.copy()
+            registry_image.registry = registry
+            self.log.debug('attempting get_manifest_digests from %s for %s',
+                           registry, registry_image)
+            manifest_list_digest = get_manifest_digests(registry_image, registry=registry,
+                                                        insecure=insecure,
+                                                        dockercfg_path=secret_path,
+                                                        versions=('list',))
+            if not manifest_list_digest.list:
+                raise PluginFailedException('no manifest list digest for %s', registry)
+            self.log.debug('Digest for registry %s is %s', registry, manifest_list_digest.list)
+            push_conf_registry = self.workflow.push_conf.add_docker_registry(registry,
+                                                                             insecure=insecure)
+            push_conf_registry.config = get_config_from_registry(registry_image, registry, None,
+                                                                 insecure, secret_path, 'v2')
```

Comment:
This will fetch the 'config' blob for the architecture=amd64 image. Are we only fetching 'config' because of the way push_conf works?

The only thing atomic-reactor does with 'config' is store an edited version of it in the Koji build metadata, and the worker builds will already have taken care of that.